android/src/main/java/ly/count/dart/countly_flutter/CountlyFlutterPlugin.java

```diff
@@ -404,6 +412,9 @@ else if ("setRequiresConsent".equals(call.method)) {
                   if (theConsent.equals("starRating")) {
                       Countly.sharedInstance().consent().giveConsent(new String[]{Countly.CountlyFeatureNames.starRating});
                   }
+                  if (theConsent.equals("performance")) {
```

Comment:
It's again using the wrong consent key. Giving and removing consent should be reworked so that it's setting them all at the same time and not one by one.